ecommerce/orders/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from baskets.models import Basket
+from core.models import BaseAbstractModel
+from customers.models import Customer, Address
+from payments.models import BankAccount
+from products.models import Product
+
+
+class BillingAddress(Address):  # Inherited from customers app model Address
+
+    class Meta:
+        verbose_name = _("billing address")
+        verbose_name_plural = _("billing addresses")
+
+    def __str__(self):
+        return super().__str__()
+
+
+class ShippingAddress(Address):  # Inherited from customers app model Address
+
+    class Meta:
+        verbose_name = _("shipping address")
+        verbose_name_plural = _("shipping addresses")
+
+    def __str__(self):
+        return super().__str__()
+
+
+class Order(BaseAbstractModel):
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE)
+    basket = models.ForeignKey(Basket, on_delete=models.PROTECT)
+    billing_address = models.ForeignKey(BillingAddress, on_delete=models.PROTECT)
+    shipping_address = models.ForeignKey(ShippingAddress, on_delete=models.PROTECT)
+    total_price = models.PositiveIntegerField(verbose_name=_("Total Price"))
```

Comment:
I think it's better to use models.DecimalField for price in OrderItem model and total_price in Order model